### PR TITLE
Pass build name when creating images

### PR DESCRIPTION
**Pull request by Akarsha-rai**
Fixes: #5405

Template `eap-cd-basic-s2i` is been removed from openshift samples as a quick fix passing build name when creating images so that template `redis-ephemeral` will be used default and will create a build

Signed-off-by: Akarsha-rai <akrai@redhat.com>

### CODE REVIEW

**Review by Shrivaibavi — APPROVED** [review]
LGTM, Please run the PR job as well

File: ocs_ci/ocs/registry.py [hunk 1]
```diff
@@ -367,19 +367,22 @@ def image_pull_and_push(project_name, template, image="", pattern="", wait=True)
...
     else:
-        cmd = f"new-app --template={template} -n {project_name}"
+        cmd = f"new-app --template={template} -n {project_name} --name=build1"
         ocp_obj = ocp.OCP()
         ocp_obj.exec_oc_cmd(command=cmd, out_yaml_format=False)
 
         # Validate it completed
         if wait:
             if template == "redis-ephemeral":
+                wait_time = 180
```
Comment (Shrivaibavi) [inline ocs_ci/ocs/registry.py:384]: Can we use wait for resource ? why 180 sec?

File: ocs_ci/ocs/registry.py [hunk 2]
```diff
@@ -367,19 +367,22 @@ def image_pull_and_push(project_name, template, image="", pattern="", wait=True)
...
+                logger.info(f"Wait for {wait_time} seconds for deplpyment to come up")
```
Comment (Shrivaibavi) [inline ocs_ci/ocs/registry.py:385]: Small typo

File: ocs_ci/ocs/registry.py [hunk 3]
```diff
@@ -367,19 +367,22 @@ def image_pull_and_push(project_name, template, image="", pattern="", wait=True)
...
             template = "redis-ephemeral"
+            image = "registry.redhat.io/rhel8/redis"
         else:
             raise
 
     if config.DEPLOYMENT.get("disconnected"):
         mirror_image(image=image)
     else:
-        cmd = f"new-app --template={template} -n {project_name}"
+        cmd = f"new-app --template={template} -n {project_name} --name=build1"
```
Comment (Shrivaibavi) [inline ocs_ci/ocs/registry.py:377]: Should we pass image as well ?

Comment (Akarsha-rai) [reply ocs_ci/ocs/registry.py:377]: no, here it is not necessary

File: ocs_ci/ocs/registry.py (hunk 1 above)
Comment (Akarsha-rai) [reply ocs_ci/ocs/registry.py:384]: no, we need the deploy/build name

**Comment by Akarsha-rai** [comment]
> PR validation on existing cluster
> 
> Cluster Name: akrai-jan31 Cluster Configuration: PR Test Suite: workloads PR Test Path: tests/e2e/workloads/ocp/registry/ Additional Test Params: OCP VERSION: 4.10 OCS VERSION: 4.10 tested against branch: master
> 
> [Job UNSTABLE](https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/2283//testReport) (some or all tests failed).

Screenshot from 2022-01-31 19-35-01

All tests failed in `sanity_helpers.health_check()` with the below error not on the exact test case
```
AssertionError: NB RPC token was not retrieved successfully within the time limit.
```

**Review by PrasadDesala — COMMENTED** [review]
LGTM but please debug the test failures in the PR verification job.

**Review by PrasadDesala — APPROVED** [review]

**Review by Shrivaibavi — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**